lib.nix

```diff
@@ -229,6 +232,8 @@ let
         outputHash = opts.sha512;
         outputHashAlgo = "sha512";
 
+        nativeBuildInputs = [ pkgs.cacert ];
```

Comment:
This is tangential to yarn 4. Newer nixpkgs disables the built-in nodejs certs so we have to supply them externally.